tool/common_mark_stats.txt

```diff
@@ -24,4 +24,4 @@
   11 of   11 – 100.0%  Tabs
    3 of    3 – 100.0%  Textual content
   19 of   19 – 100.0%  Thematic breaks
- 561 of  624 –  89.9%  TOTAL
+ 558 of  624 –  89.4%  TOTAL
```

Comment:
ooo...regression here!

Reply:
Gotta get rid of these regressions...

Reply:
Whoops, I was only generating the test files for GFM